src/senselab/text/tasks/evaluate_conversation/deep_eval.py

```diff
@@ -0,0 +1,33 @@
+"""deep_eval.py."""
+
+from typing import Dict, List
+
+from senselab.utils.data_structures.script_line import ScriptLine
+
+from .metrics import Rouge
+
+
+def evaluate_conversation(script_lines: List[ScriptLine], metrics: List[str]) -> Dict:
+    """Evaluate a conversation based on the provided script lines and metrics.
+
+    Args:
+        script_lines (List[ScriptLine]): A list of script lines to evaluate.
+        metrics (List[str]): A list of metrics to use for evaluation.
```

Comment:
Is `metrics` ever used?

Reply:
Deep eval is legacy and should have been deleted but it must have copied over when I updated my branch